redisgraph/client.py

```diff
@@ -116,7 +116,7 @@ def query(self, q):
         response = self.redis_con.execute_command("GRAPH.QUERY", self.name, q)
         data = response[0]
         statistics = response[1]
-        result_set = [res.split(',') for res in data]
+        result_set = [res.decode().split(',') for res in data]
```

Comment:
Why do we need decode here?

Reply:
Looks like an artefact of trying to track down the issue that can be removed.